types/RoundWinReason.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum RoundWinReason {
+  PlayersKilled,
+  BombDefused,
+  BombExploded,
+}
```

Comment:
Time up

Reply:
Maybe edge case, but what if they surrender and the game ends? What happens with that round? Or doesn't this exist since it's always at the end of the round or before the round?

Reply:
Hmm that would probably be a game end reason, not a round end reason. Would have to double check in demo though